bmaptools/BmapHelpers.py

```diff
@@ -83,3 +94,72 @@ def program_is_available(name):
             return True
 
     return False
+
+
+def get_mount_point(filepath):
+    """
+    Return the mount point the file object file resides on.
```

Comment:
"the file object file" - not very well phrased. The input is path, the output is the mount point. I suggest:

Return the mount point for path in 'filepath'.

Also, there is not reason to limit this to files, it'll work for non-files too, right? Then just call the variable 'path'.